sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_path_client.py

```diff
@@ -204,6 +218,28 @@ def _create(self, resource_type, content_settings=None, metadata=None, **kwargs)
             For example, if p is 0777 and u is 0057, then the resulting permission is 0720.
             The default permission is 0777 for a directory and 0666 for a file. The default umask is 0027.
             The umask must be specified in 4-digit octal notation (e.g. 0766).
+        :keyword str owner:
+            The owner of the blob or directory. Default value is None.
+        :keyword str group:
+            The owning group of the blob or directory. Default value is None.
+        :keyword str acl:
+            Sets POSIX access control rights on files and directories. The value is a
+            comma-separated list of access control entries. Each access control entry (ACE) consists of a
+            scope, a type, a user or group identifier, and permissions in the format
+            "[scope:][type]:[id]:[permissions]". Default value is None.
+        :keyword str lease_id:
+            Lease ID, in a GUID string format. The Blob service returns
+            400 (Invalid request) if the lease ID is not in the correct format. See Guid
+            Constructor (String) for a list of valid GUID string formats. Default value is None.
+        :keyword int lease_duration:
+            The lease duration is required to acquire a lease, and specifies the
+            duration of the lease in seconds.  The lease duration must be between 15 and 60 seconds or -1
+            for infinite lease. Default value is None.
+        :keyword expiry_options:
+            Indicates mode of the expiry time. Default value is None.
+        :paramtype expiry_options: str or ~azure.storage.filedatalake.models.PathExpiryOptions
+        :keyword str expires_on:
```

Comment:
The type here should be `datetime`. I believe we already do the conversion correctly as previously discussed. However, if we support the RelativeTo* options above we also need to except an `int` here as it would be number of milliseconds from the create/now time. You can see this and pull the docstring from [`set_file_expiry`](https://github.com/Azure/azure-sdk-for-python/blob/01108413a17bfc9f75c2c571fbb6799f1cce6867/sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_file_client.py#L284).